demo/demo-etcd/test-client/src/main/java/org/apache/servicecomb/samples/EtcdConfigIT.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.servicecomb.samples;
+
+import java.nio.charset.StandardCharsets;
+
+import org.apache.servicecomb.demo.CategorizedTestCase;
+import org.apache.servicecomb.demo.TestMgr;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestOperations;
+import org.springframework.web.client.RestTemplate;
+
+import io.etcd.jetcd.ByteSequence;
+import io.etcd.jetcd.Client;
+
+@Component
+public class EtcdConfigIT implements CategorizedTestCase {
+  RestOperations template = new RestTemplate();
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(EtcdConfigIT.class);
+
+  @Override
+  public void testRestTransport() throws Exception {
+
+    testEnvironment();
+    testApplication();
+    testService();
+    testVersion();
```

Comment:
Can you add a test case for priority? e.g. service configuration override application configuration